services/compute/virtual-machines/taxonomy.md

```diff
@@ -0,0 +1,47 @@
+|	CCC-030117	|	Instance Templates	|	Offering templates for provisioning virtual machine instances with pre-configured images, predefined configurations and settings and/or bootstrap scripts.	|
+|	CCC-030118	|	Instance Preemptibility	|	Providing the option for using preemptible virtual machine (spot) instances at a lower cost for non-critical or fault-tolerant workloads that may be terminated by the cloud provider after a notice period.	|
+|	CCC-030119	|	Instance Affinity/Anti-affinity	|	Enabling control over the location of virtual machine instances to ensure or prevent co-location on the same physical hardware.	|
+|	CCC-030120	|	Instance Health Checks	|	Providing features for performing health checks on virtual machine instances and automatically replacing or repairing unhealthy instances.	|
+|	CCC-030121	|	Instance Remote Access	|	Offering remote access to virtual machine instances through methods such as SSH or RDP for troubleshooting, debugging, and maintenance purposes.	|
+|	CCC-030122	|	Instance Live Migration	|	The ability to perform live migration of virtual machine instances between physical hosts for maintenance or load balancing purposes without downtime.	|
+|	CCC-030123	|	Instance Remote Configuration	|	Providing tools for remotely configuring virtual machine instances, including deployment automation and configuration management frameworks.	|
+|	CCC-030124	|	Instance Resource Tagging	|	Enabling tagging of virtual machine instances with metadata for organization, management, and cost allocation purposes.	|
+|	CCC-030125	|	Instance Resource Monitoring	|	Providing tools for monitoring resource utilization and performance metrics for virtual machine instances, including CPU usage, memory usage, disk I/O, and network traffic.	|
+|	CCC-030126	|	Custom Images	|	Allows users to create and manage their own customized virtual machine images.	|
+|	CCC-030127	|	Dynamic Performance	|	Providing "burstable" instances for intermittent workloads that accumulate credits during periods of low usage which can be used to burst above baseline performance when needed.	|
+|	CCC-030128	|	Dedicated Instances	|	Providing the option to run instances on physical servers that are dedicated solely to a single customer account, ensuring that the underlying hardware resources are not shared with other customers.	|
+|	CCC-030129	|	Instance Scheduling	|	Offering features for scheduling the startup, shutdown, and maintenance of virtual machine instances based on predefined schedules or conditions.	|
+|	CCC-030130	|	Interoperability with Storage Options	|	Capability to read/write to non-ephemeral external storage including object storage and encrypted block storage.	|
+|	CCC-030131	|	Instance Autoscaling	|	Automatically adjusting the number and instance type of virtual machine instances based on predefined criteria such as CPU utilization or incoming traffic,	|
```

Comment:
Done by Auto Scaling Group in the other hand VM should provide a start/stop API interface for the Auto Scaling Group to work on, which is a feature of the VM.

Reply:
I've removed the vertical and horizontal scaling as features, and addressed providing an interface for the autoscaling services in a separate point.